Jellyfin.Windows.Tray/Properties/Resources.resx

```diff
@@ -119,6 +119,6 @@
   </resheader>
   <assembly alias="System.Windows.Forms" name="System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
   <data name="JellyfinIcon" type="System.Resources.ResXFileRef, System.Windows.Forms">
-    <value>..\JellyfinIcon.ico;System.Drawing.Icon, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
+    <value>..\Resources\JellyFinIcon.ico;System.Drawing.Icon, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
```

Comment:
```suggestion
    <value>..\Resources\JellyfinIcon.ico;System.Drawing.Icon, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
```

The `f` should be lowercase

Reply:
I caught that before I submitted.  I just checked and my branch does have the lower case "f"

Reply:
I think something went wrong as the file itself is still named "JellyFinIcon.ico" and the resx file still points to that name.

[This Stack Overflow question might help](https://stackoverflow.com/a/24979063)

Reply:
OK, I think I caught all the capital "F" instances in my latest commit.  Sorry about that.

Reply:
Unfortunately the file still got the wrong name, see [here](https://github.com/jellyfin/jellyfin-server-windows/pull/26/files#diff-83163a142865202d1c729e3ef871cd4c)

Reply:
Maybe third times the charm?  I renamed it in Git and pushed.

Reply:
Yup, the file naming looks good now

Reply:
Thank you for the help!!